introducing-signal-store.md

```diff
@@ -0,0 +1,701 @@
+Welcome to **MiniRx Signal Store**, the new state management library from [MiniRx](https://mini-rx.io/).
+
+With the arrival of [Signals](https://angular.io/guide/signals) in Angular 16, new best practices are arising for state and events. In the future Signals and Observables will coexist in your application.
+
+Modern Angular needs modern state management which promotes **new Angular best practices** and **streamlines the usage of Signals and Observables**...
+
+MiniRx Signal Store is exactly that.
+
+### Modern state management with MiniRx Signal Store:
+
+* Signal Store is an **[Angular](https://angular.dev/)-only** state management library
+* Signal Store **embraces [Angular Signals](https://angular.io/guide/signals)** and leverages **Modern Angular APIs** internally
+* Signal Store is based on the same great concept as the original **[MiniRx Store](https://mini-rx.io/)**
+  * Manage **global** state at large scale with the **Store (Redux) API**
+  * Manage **global** state with a minimum of boilerplate using **Feature Stores**
+  * Manage **local** component state with **Component Stores**
+  * All combined in a single library
+  * MiniRx always tries to find the sweet spot between powerful, simple and [lightweight](https://github.com/spierala/angular-state-management-comparison)
+* Signal Store implements and promotes new **Angular best practices**:
+  * **Signals** are used for **(synchronous) state**
+  * **RxJS** is used for events and **asynchronous tasks**
+* Immutable Signal State: Immutability can be enforced with the Immutable State extension
+* Signal Store helps to streamline your usage of [RxJS](https://rxjs.dev/) and [Signals](https://angular.io/guide/signals): e.g. `connect` and `rxEffect` understand both Signals and Observables
+* Signal Store has first-class support for OOP style (e.g. `MyStore extends FeatureStore`), but offers also functional creation methods (e.g. `createFeatureStore`)
+* Simple refactor: If you used MiniRx Store before, refactor to Signal Store will be straight-forward: change the TypeScript imports, remove the Angular async pipes (and ugly non-null assertions (`!`)) from the template
+
+### Getting Started
+To install the @mini-rx/signal-store package, use your package manager of choice:
+
+`npm install @mini-rx/signal-store`
+
+### Use-cases
+
+MiniRx Signal Store is highly flexible and offers three different well-defined state containers out of the box:
```

Comment:
`MiniRx Signal Store` - that's what I meant earlier - this one is 100% clear.

Reply:
Yes, I will be more consistent with mentioning **MiniRx** Signal Store to make it more clear.